From 4da914d1d7e17deccf37e67bbff268768a02a42f Mon Sep 17 00:00:00 2001
From: Abderrahim Kitouni <akitouni@gnome.org>
Date: Sat, 4 May 2024 12:28:44 +0100
Subject: [PATCH] components/linux.bst: strictly depend on linux-module-cert

If the certificate changes, we need to rebuild. Otherwise, the kernel
will refuse to load modules signed with the new key.
---
 elements/components/linux.bst | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/elements/components/linux.bst b/elements/components/linux.bst
index 8f4818d39..6df78ad47 100644
--- a/elements/components/linux.bst
+++ b/elements/components/linux.bst
@@ -11,8 +11,9 @@ build-depends:
 - components/bc.bst
 - components/gzip.bst
 - components/rsync.bst
-- components/linux-module-cert.bst
 - components/util-linux.bst
+- filename: components/linux-module-cert.bst
+  strict: true
 
 variables:
   bootdir: /boot
-- 
2.43.0

